main.js

```diff
@@ -0,0 +1,35 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+renderer.renderPosts(tweeter.getPosts())
+const twit = $('#post')
+const inputPost = $('input')
```

Comment:
I would call this `postInput`. Also, when a variable has a jQuery element inside of it, it is good practice to add a `$` at the beginning of the name like: `const $postInput = $('input')`